packages/create-sitecore-jss/src/templates/nextjs-sxa/src/pages/_app.tsx

```diff
@@ -1,9 +1,9 @@
 import type { AppProps } from 'next/app';
 import { I18nProvider } from 'next-localization';
 
-import 'bootstrap/dist/css/bootstrap.css';
-import 'assets/sass/main.scss';
-import 'assets/basic/main.scss';
+// in the main.css is the list of used css
```

Comment:
Here should be `scss` and do we need to mention that `main.scss` contains CSS? It's expected

Reply:
I agree)